test/example_app_test.exs

```diff
@@ -0,0 +1,85 @@
+defmodule ExampleAppTest do
+  use ExampleApp.DataCase
+  alias ExampleApp.Users.User
+  alias ExampleApp.ObjectTypes.ProjectManager
+
+  setup do
+    data = [
+      %{
+        "ID" => "5ba94a2a00854529705f809ebec755b9",
+        "emailAddr" => "user1@test.com"
+      },
+      %{
+        "ID" => "5b93028601bd041b925dc2067422be82",
+        "emailAddr" => "user2@test.com"
+      },
+      %{
+        "ID" => "5b89e3d300abc77aa4885e819fe3713d",
+        "emailAddr" => "user3@test.com"
+      }
+    ]
+
+    Enum.each(data, fn %{"ID" => wf_user_id, "emailAddr" => email} ->
+      Repo.insert!(%User{
+        wf_user_id: wf_user_id,
+        email: email
+      })
+    end)
+
+    data = [
+      %{
+        "ownerID" => "5ba94a2a00854529705f809ebec755b9"
+      },
+      %{
+        "ownerID" => "5b93028601bd041b925dc2067422be82"
+      },
+      %{
+        "ownerID" => "5b89e3d300abc77aa4885e819fe3713d"
+      }
+    ]
+
+    Enum.each(data, fn %{"ownerID" => wf_user_id} ->
+      Repo.insert!(%ProjectManager{
+        wf_user_id: wf_user_id
+      })
+    end)
+  end
+
+  test "it works" do
```

Comment:
added a test to show that it should be working now, `mix test test/example_app_test.exs`

Reply:
note that you might need to recreate the database, since I've updated the migrations in place.

```
mix do ecto.drop, ecto.create, ecto.migrate
MIX_ENV=test mix do ecto.drop, ecto.create, ecto.migrate
```